test/Microsoft.AspNetCore.Mvc.TagHelpers.Test/FormTagHelperTest.cs

```diff
@@ -26,6 +26,64 @@ namespace Microsoft.AspNetCore.Mvc.TagHelpers
     public class FormTagHelperTest
     {
         [Fact]
+        public async Task ProcessAsync_OnlyPageHanlerGeneratesExpectedOutput()
```

Comment:
> You could write this test like ProcessAsync_InvokesGeneratePageForm. We just want to make sure we call the IHtmlGenerator with the right params here.

Sure, there's already a test like that for `asp-page-handler` + other attributes, i'll re-purpose this one to just do page handler.